tsconfig.json

```diff
@@ -19,9 +19,6 @@
 	],
 	"exclude": [
 		"**/node_modules",
-		"**/build",
-		"**/bin",
-		"**/out",
```

Comment:
Not that I really care but why were these removed?

Reply:
Those are not used anywhere, we only use `dist`, so no reason for those to exist.